businesscentral-monitoring/image.yaml

```diff
@@ -48,6 +48,17 @@ envs:
     - name: "KIE_SERVER_CONTROLLER_TOKEN"
       example: "dasd373egds"
       description: "KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)"
+## OpenShift Enhancement BEGIN
+    - name: KIE_CONTROLLER_OCP_ENABLED
+      example: "true"
+      description: "If set to true, turns on OpenShift integration feature (Sets the org.kie.workbench.controller.openshift.enabled system property)"
+    - name: KIE_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE
```

Comment:
This option comes with a drawback. Execution servers page of Workbench shows URLs of Kie servers and their containers - so users can simply access Kie server URL. In case this property is enabled the URLs point to internal addresses, unavailable for users.
Wouldn't it have more sense to set this property to false as default?

Reply:
Agree. Change it to 'false'. Is there a feature that BC/WB will recognize an internal service endpoint url and hide it from the UI subsequently?

Reply:
I think there is such feature, not sure if it is enabled for both Kie server URL and container URL.

Reply:
yes, if the BC can't access the KIE Server url it will be hide from console, IIRC this function tries to validate external access simulating a user. An internal address will probably not be shown there.